class.jetpack.php

```diff
@@ -4844,6 +4845,24 @@ function build_connect_url( $raw = false, $redirect = false, $from = false, $reg
 		return $raw ? $url : esc_url( $url );
 	}
 
+	/**
+	 * Get the site age, based on the earliest user registration date.
+	 *
+	 * @since 7.1.0
+	 *
+	 * @return string Site creation date and time.
+	 */
+	public static function get_site_creation_date() {
```

Comment:
This has enough chance to be a misnomer (related to @kraftbj's comment) that I would recommend we rename it explicitly to `get_earliest_user_registration_date()`